src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard.module.ts

```diff
@@ -7,9 +7,10 @@ import { NgbNavModule, NgbPopoverModule } from '@ng-bootstrap/ng-bootstrap';
 import { ChartsModule } from 'ng2-charts';
 
 import { SharedModule } from '~/app/shared/shared.module';
+import { NewDashboardModule } from '../new-dashboard/new-dashboard.module';
```

Comment:
My idea on naming this component as `NewDashboardModule` was that this one is ment to replace the current `DashboardModule` so when the current dashboardmodule would be deprecated, I would rename `NewDashboardModule` to `DashboardModule` and get rid of the deprecated one.

I can see the concern of this naming, but I believed the goal is not to maintain both dashboards but rather have only one of them in the future. 

Is this still a bad practice @epuertat ? I can just rename the component as `DashboardV3Component` as you mentioned if this naming it is not appropiate.

Reply:
Yesterday I was sharing with @nizamial09 [I quote that I recently read](https://martinfowler.com/bliki/TwoHardThings.html): "There are only 2 hard things in Computer Science: cache invalidation, and naming things".

The main issue I see with calling things "New", "Old", etc, is that they are imprecise and temporary/subject to change (what is "New" today might become "Old" tomorrow). On the other hand, "v2" or "DashboardWithPrometheusTimeseries" define components univocally. The present-future thing is relevant here because we don't only have to take care of today's code, but also for several years (depending on the release support lifecycle), and we may end up seeing multiple repetitions of the New/Old Dashboard names.

Reply:
Renamed to `DashboardComponentV3`